reconcile/test/test_openshift_cluster_bots.py

```
@@ -159,33 +241,272 @@ def test_retrieve_token(mocker: MockerFixture) -> None:
+    vault_mock = mocker.patch(
+        "reconcile.openshift_cluster_bots.VaultClient.get_instance"
+    )
+
+    result = ocb.process_entry("kubeconfig", c, config, entry, cluster_admin=False)
+
+    assert result.action == "skipped"
+    assert oc_mock.call_count == 1
+    vault_mock.return_value.delete.assert_not_called()
 
 
 class Mocks:  # ruff: ignore[class-as-data-structure]
```

Ruff (0.16.1)

[warning] Invalid rule code in suppression: class-as-data-structure

Enable `lint.preview` to use rule names

(RUF102)